client/landing/gutenboarding/components/signup-form/style.scss

```diff
@@ -63,6 +63,18 @@
 		}
 	}
 
+	.signup-form__fieldset {
+		border: 0;
+		font-family: inherit;
+		font-size: 100%;
```

Comment:
Seems like many (or all) of those font styles aren't necessary, are they? I think they were there only because this was part of a global reset before.

Reply:
You're right, I've removed those font related styles. Thanks!